.github/workflows/integration-tests-kint.yml

```diff
@@ -4,6 +4,7 @@ on:
     pull_request:
         branches:
             - master
+            - brendon/feat-add-foreign-asset-model
```

Comment:
I'll remove this in a quick throwaway PR once this work has been merged successfully.